katalog/prometheus-operator/README.md

```diff
@@ -39,14 +39,14 @@ illustrated in this image from Prometheus Operator repository:
 
 ## Requirements
 
-- Kubernetes >= `1.21.0`
+- Kubernetes >= `1.23.0`
```

Comment:
```suggestion
- Kubernetes >= `1.24.0`
```

Maybe the same applies here

Reply:
mental note: we'll need to keep this in mind for the next releases of KFD, we can't update monitoring for KFD 1.23.x